apps/gov-website/pages/proposal/new.tsx

```diff
@@ -69,12 +69,30 @@ const NewProposal: NextPage<NewProposalProps> = ({ extrinsics }) => {
 			<Layout.PageContent>
 				<Layout.PageHeader>Submit a Proposal</Layout.PageHeader>
 
-				<p className="prose mb-[1.5em] text-lg">
-					To submit a proposal you must be a CENNZnet Councillor. Lorem laborum
-					dolor minim mollit eu reprehenderit culpa dolore labore dolor mollit
-					commodo do anim incididunt sunt id pariatur elit tempor nostrud nulla
-					eu proident ut id qui incididunt.
-				</p>
+				<div className="mb-[1.5em] space-y-2">
+					<p className="prose text-lg">
+						Councillors are allowed to submit proposals, which are made up of
+						three sections.
+					</p>
+
+					<ol className="prose -ml-2 list-decimal text-lg">
```

Comment:
Make the text a bit smaller plus add more indentation so it reads better

```suggestion
					<ol className="prose ml-8 list-decimal text-xs">
```